back-end/db/player.ts

```diff
@@ -141,6 +141,37 @@ export default {
     }
   },
 
+  awardAllPlayersConsolationPoints: async function(guessingPlayers: IPlayer[], quizQIndex: number): Promise<any> {
+    const guessTime = p => p.quizGuesses[quizQIndex].timestamp;
+    const timeBetweenPlayerGuesses = (a: IPlayer, b: IPlayer) => guessTime(b) - guessTime(a);
+
+    guessingPlayers.sort(timeBetweenPlayerGuesses);
```

Comment:
Sort the players by their guess times - whoever answered fastest will be last in the list